### PR TITLE
feat: add 'prev' and 'next' version navigation to downloads/releases

### PR DESCRIPTION
**Pull request by Nnyny**
fixes: https://github.com/keymanapp/keyman.com/issues/392

### CODE REVIEW

File: downloads/releases/_version_downloads.php [hunk 1]
```diff
@@ -37,24 +37,86 @@ function array_key_first(array $arr) {
...
 
 <?php
   if(empty($versions->android)) {
-    echo "<p>Version $version may be invalid, or may not yet be available.</p>";
+    echo "<p>Version $currentVersion may be invalid, or may not yet be available.</p>";
     exit;
   }
+?>
+
+<a class ="button" href='<?=$KeymanHosts->help_keyman_com?>/version-history/all-versions.php#<?=$versionNumber?>'>View version's history</a>
```
Comment (darcywong00) [inline downloads/releases/_version_downloads.php:116]: On my development, I was at http://keyman.com.localhost/downloads/releases/14.0.209 so the button had this link:

http://help.keyman.com.localhost/version-history/all-versions.php#14-0-209-beta

But it didn't work for me. 

Is there another PR I need to bring in for help.keyman.com?

Comment (Nnyny) [reply downloads/releases/_version_downloads.php:116]: Yes, I'll do another PR for help.keyman.com.

**Review by mcdurdin — APPROVED** [review]
LGTM, @darcywong00 do you concur? (I haven't tested locally)

**Comment by darcywong00** [comment]
Yep - will merge

**Pull request merged**